packages/amplify-category-geo/src/service-utils/placeIndexUtils.ts

```diff
@@ -52,7 +62,8 @@ export const modifyPlaceIndexResource = async (context: $TSContext, parameters:
   const placeIndexStack = new PlaceIndexStack(new App(), 'PlaceIndexStack', { ...parameters, ...templateMappings, authResourceName });
   generateTemplateFile(placeIndexStack, parameters.name);
   saveCFNParameters(parameters);
-  stateManager.setResourceInputsJson(pathManager.findProjectRoot(), category, parameters.name, { groupPermissions: parameters.groupPermissions });
+  stateManager.setResourceInputsJson(pathManager.findProjectRoot(),
+    category, parameters.name, { groupPermissions: parameters.groupPermissions });
```

Comment:
There should be a function to take care of saving state. Most of this code is duplicated 5 times